test/inductor/test_debug_trace.py

```diff
@@ -51,20 +56,27 @@ def fn(a, b):
             with self.assertLogs(
                 logging.getLogger("torch._inductor.debug"), level=logging.WARNING
             ) as cm:
-                fn(torch.randn(16, 16), torch.randn(16, 16))
+                with ctx():
+                    fn(torch.randn(16, 16), torch.randn(16, 16))
```

Comment:
```suggestion
            ) as cm, ctx():
                fn(torch.randn(16, 16), torch.randn(16, 16))

Reply:
Thanks, done in [255b0f1](https://github.com/pytorch/pytorch/commit/255b0f1afe48e27358d3fb39f20961afdc5ab0dd)